<launch>
	<!-- ROSCORE automatically started if not already running -->
	<!-- teleoperation node -->
	<node name="base_teleoperation" pkg="base_teleop" type="base_teleop"/>
	<!-- joy node -->
	<node respawn="true" pkg="joy" type="joy_node" name="base_joy" >
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.12" />
	</node>
	
	<!-- Axes -->
	<param name="axis_left" value="1" type="int"/>
	<param name="axis_right" value="4" type="int"/>
	<param name="scale_linear" value="2" type="double"/>
	<param name="scale_angular" value="2" type="double"/>
</launch>
